test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Routing/DefaultODataPathHandlerTest.cs

```diff
@@ -659,7 +659,6 @@ public void CanParsePropertySegment(string odataPath)
             PropertySegment propertySegment = Assert.IsType<PropertySegment>(segment);
 
             Assert.Equal(expectedText, propertySegment.ToUriLiteral());
-            Assert.Null(path.NavigationSource);
```

Comment:
>Assert.Null(path.NavigationSource); [](start = 12, length = 35)

not only remove it, try to verify it with the previous navigation source.

Reply:
Same for others?

---
In reply to: [279473187](https://github.com/OData/WebApi/pull/1738#discussion_r279473187) [](ancestors = 279473187)